docs/Providers.md

```diff
@@ -0,0 +1,72 @@
+Providers
+=========
+
+When using Moya, you make all API requests through a `MoyaProvider` instance, 
+passing in a value of your enum that specifies which endpoint you want to call. 
+After setting up your [Endpoint](Endpoints.md), you're basically all set for
+basic usage:
+
+```swift
+let GitHubProvider = MoyaProvider<GitHub>()
+```
+
+After that simple setup, you're off to the races:
+
+```swift
+provider.request(.Zen) { (data, statusCode, response, error) in
+    if let data = data {
+        // do something with the data
+    }
+}
+```
+
+That's it! The `request()` method returns a `Cancellable`, which has
+only one public function, `cancel()`, which you can use to cancel the
+request. 
+
+Remember, *where* you put your target and the provider, are completely up 
+to you. You can check out [Artsy's implementation](https://github.com/Artsy/eidolon/Kiosk/AppNetworking/ArtsyAPI.swift)
```

Comment:
URL is invalid. Should be `https://github.com/artsy/eidolon/blob/master/Kiosk/App/Networking/ArtsyAPI.swift`